packages/flutter/lib/src/rendering/wrap.dart

```diff
@@ -86,9 +86,40 @@ class _RunMetrics {
   final int childCount;
 }
 
+/// How the child is inscribed into the available space.
+///
+/// See also:
+///
+///  * [RenderWrap], the wrap render object.
+///  * [Wrap], the wrap widget.
+///  * [Wrapped], the widget to set the [WrapFit].
+enum WrapFit {
+  /// The child is forced to fill the available space in the current run, if
+  /// its min intrinsic size in the [Wrap.direction] allows it.
+  ///
+  /// The [Wrapped] widget assigns this kind of [WrapFit] to its child.
+  runTight,
```

Comment:
here and below: We should probably include a statement about how this is somewhat expensive, similarly to what's document on https://main-api.flutter.dev/flutter/widgets/IntrinsicHeight-class.html (last paragraph).

Reply:
I don't think it's that bad, since it only computes the minimal size of the child which should be linear to the size of the sub tree. Therefore I've added this paragraph: 8a605c96dc91b6f0ddd665dae14b8aec41d4ee20